podcasts.xcodeproj/project.pbxproj

```diff
@@ -11219,7 +11235,10 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIconPrototype;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_ENTITLEMENTS = podcasts/podcasts.prototype.entitlements;
+				CODE_SIGN_ENTITLEMENTS = podcasts/podcastsDebug.entitlements;
```

Comment:
Just wondering if these changes are Xcode stuff or something else?

Reply:
I _think_ they were Xcode but I went in and undid them and Xcode hasn't added them back yet.